masonry_core/src/core/action.rs

```diff
@@ -15,7 +15,7 @@ use crate::core::PointerButton;
 /// Note: Actions are still a WIP feature.
 pub enum Action {
     /// A button was pressed.
-    ButtonPressed(PointerButton),
+    ButtonPressed(Option<PointerButton>),
```

Comment:
This needs an explanation in the doc comment as to what it means if this is `None`.